src/chainparams.cpp

```diff
@@ -202,12 +203,12 @@ class CMainParams : public CChainParams {
 
         nPoolMaxTransactions = 3;
         nFulfilledRequestExpireTime = 60 * 60; // fulfilled requests expire in 1 hour
-        strSporkPubKey = "048459e70138ccb109b38de57aa87b5b49b1b7c92550ee29b25e8eca195063d8872cc256bb35688ff6159112b802989f4b87a87d5ee1d6c1747c6e4bcdb68a3dae";
+        strSporkPubKey = "";
```

Comment:
We keep deleting the spork pub key and if we don't have one, we will not be able to spork.  Same with the alert pub key.